features/step_definitions/wait_steps.rb

```diff
@@ -53,30 +59,57 @@
 end
 
 When('I wait for the section element that takes a while to disappear') do
-  @test_site.section_experiments.removing_parent_section.wait_for_no_removing_section_element
+  @test_site
+    .section_experiments
+    .removing_parent_section
+    .wait_for_no_removing_section_element
 end
 
 Then('the slow section appears') do
-  expect(@test_site.section_experiments.parent_section).to have_slow_section_element
+  expect(
+    @test_site.section_experiments.parent_section
+  ).to have_slow_section_element
 end
 
 Then(/^the removing section disappears$/) do
-  expect(@test_site.section_experiments.removing_parent_section).not_to have_removing_section_element
+  expect(
+    @test_site.section_experiments.removing_parent_section
+  ).not_to have_removing_section_element
 end
 
 Then(/^an exception is raised when I wait for a section that won't appear$/) do
-  expect { @test_site.section_experiments.parent_section.wait_for_slow_section_element(1) }
-    .to raise_error(SitePrism::TimeOutWaitingForExistenceError)
-    .with_message('Timed out after 1s waiting for Parent#slow_section_element')
-end
-
-Then(/^an exception is raised when I wait for a section that won't disappear$/) do
-  expect { @test_site.section_experiments.removing_parent_section.wait_for_no_removing_section_element(1) }
-    .to raise_error(SitePrism::TimeOutWaitingForNonExistenceError)
-    .with_message('Timed out after 1s waiting for no Parent#removing_section_element')
-end
-
-When(/^I wait for the collection of sections that takes a while to disappear$/) do
+  expect do
+    @test_site
+      .section_experiments
+      .parent_section
+      .wait_for_slow_section_element(1)
+  end.to(
+    raise_error(SitePrism::TimeOutWaitingForExistenceError)
+      .with_message(
+        'Timed out after 1s waiting for Parent#slow_section_element'
+      )
+  )
+end
+
+Then(
+  /^an exception is raised when I wait for a section that won't disappear$/
+) do
+  expect do
```

Comment:
Same here. Also whilst you're here can you convert the regex syntax to string syntax